MobileBuy/buy/src/androidTest/java/com/shopify/buy/service/BuyTest.java

```diff
@@ -226,13 +226,30 @@ public void testFetchingShippingRatesWithInvalidCheckout() throws InterruptedExc
     public void testCheckoutFlowUsingCreditCard() throws InterruptedException {
         createValidCheckout();
         fetchShippingRates(HttpStatus.SC_OK);
-        updateCheckout();
+        setShippingRate();
         addCreditCardToCheckout();
         completeCheckout();
         pollCheckoutCompletionStatus();
         getCheckout();
     }
 
+    public void testChangedShippingAddress() throws InterruptedException {
+        createValidCheckout();
+        fetchShippingRates(HttpStatus.SC_OK);
+        setShippingRate();
+
+        Address address = checkout.getShippingAddress();
+        address.setCity("Toronto");
+
```

Comment:
It is not entirely necessary, but I would like to have it in place to match the rest of the integration functions in this file.